packages/auth/src/interaction/routes.test.ts

```diff
@@ -41,7 +47,7 @@ describe('Interaction Routes', (): void => {
   let grant: Grant
 
   const generateBaseGrant = () => ({
-    state: GrantState.Pending,
+    state: GrantState.Processing,
```

Comment:
Are the "base" grants in the tests supposed to reflect the state of a grant when its created or does it not really matter? That was my impression but not sure. If so, I think the `BASE_GRANT`/`generateBaseGrant()` states could be updated to `Processing` (from `Pending`) in these files:

access/service.test.ts
accessToken/routes.test.ts
accessToken/service.test.ts
grant/routes.test.ts

I changed a few and re-ran and it doesn't look like it causes any failures.

Reply:
Before this PR I would say that base grants should reflect its state when it's created. Your question though, did get me thinking about if these base grants ought to reflect the state the grant _should_ be in when the tests are happening (e.g. should test grant states reflect when they are created for `access` tests or reflect when they're approved for `accessToken` tests?).

Fortunately with this PR implementing the `Processing` state, it can be used in tests as a sort of "generic" state. A grant would be processing when it is created and may be processing if it was previously approved, but the client modified the grant after that fact.